content/03.combining-datasets.md

```diff
@@ -1,23 +1,28 @@
-### Constructing machine learning-ready rare disease datasets
+## Constructing machine learning-ready rare disease datasets
 
-High-throughput 'omic' data methods generate high-dimensional data or data with many features, regardless of the underlying disease or condition being assayed.
+High-throughput 'omic' data methods generate high-dimensional data (or data containing many features), regardless of the underlying disease or condition being assayed.
 A typical rare disease dataset is comprised of a small number of samples [@doi:10.1186/s13023-020-01424-6].
-A lack of samples gives rise to the “curse of dimensionality” (i.e., few samples but many features), which can contribute to the poor performance of models [@doi:10.1038/nrc2294] [TODO: reference new figure as appropriate #186].
-More features often means increased missing observations (_sparsity_), more dissimilarity between samples (_variance_), and increased redundancy between individual features or combinations of features (_collinearity_) [@doi:10.1038/s41592-018-0019-x], all of which contribute to a challenging prediction problem.
+A lack of samples gives rise to the “curse of dimensionality” (i.e., few samples but many features), which can contribute to the poor performance of models [@doi:10.1038/nrc2294] (Figure {@fig:1}a).
```

Comment:
Citing Fig 1A (assuming I'm in the right place with https://github.com/jaybee84/ml-in-rd/blob/a7c10073261bed4c95db1bebfe407b7f9ac96784/content/images/figures/pdfs/figure-1-combining-datasets.png) seems a bit odd here.

Reply:
I was still envisioning our heatmap figure here for some reason :(